core/services/ocr2/delegate.go

```diff
@@ -1135,14 +1137,33 @@ func (d *Delegate) newServicesOCR2Keepers20(
 		d.cfg.JobPipeline().MaxSuccessfulRuns(),
 	)
 
-	return []job.ServiceCtx{
+	automationServices := []job.ServiceCtx{
 		job.NewServiceAdapter(runr),
 		runResultSaver,
 		keeperProvider,
 		rgstry,
 		logProvider,
 		pluginService,
-	}, nil
+	}
+
+	if d.cfg.OCR2().CaptureAutomationCustomTelemetry() {
```

Comment:
did we discuss whether TOML config would be a better place for this config? cc @anirudhwarrier